.gitignore

```diff
@@ -1,2 +1,2 @@
-.idea/
```

Comment:
Don't remove this please, otherwise IDE settings for PHPStorm will get committed for every project. IntelliJ/family is the default IDE we use as BC.